docs/source/getting-started/networking-basics.md

```diff
@@ -35,10 +34,20 @@ Configuring only the main IP and port of JupyterHub should be sufficient for
 most deployments of JupyterHub. However, more customized scenarios may need
 additional networking details to be configured.
 
-Note that `c.JupyterHub.ip` and `c.JupyterHub.port` are single values,
+Note that `c.JupyterHub.bind_ip` and `c.JupyterHub.port` are single values,
 not tuples or lists – JupyterHub listens to only a single IP address and
 port.
 
+**Deprecated in 0.9:** Alternatively, Proxy's main IP address and port can be set by placing the following lines in the configuration file:
+
+```python
+c.JupyterHub.ip = '192.168.1.2'
+c.JupyterHub.port = 443
+```
+
+We strongly recommend using the `bind_ip` setting instead of the deprecated
+settings, `ip` and `port`.
```

Comment:
```suggestion
We recommend using the `bind_url` setting instead of `ip` and `port`,
since `bind_url` can set ip, port, and base_url in one place.
```

Reply:
agree with @minrk correction